scripts/classes/class.Extrausers.php

```diff
@@ -53,12 +53,21 @@ private static function _generateFile($file, $query, &$cronlog)
 			switch ($type) {
 				case 'passwd':
 					$line = $u['username'] . ':' . $u['password'] . ':' . $u['uid'] . ':' . $u['gid'] . ':' . $u['comment'] . ':' . $u['homedir'] . ':' . $u['shell'] . PHP_EOL;
+					$permissions = 0644;
+					$user = 'root';
```

Comment:
user is always root, also hardcoded ...as this script is run as cron with user root, the owner and group of newly created files should be the root user anyway...superfluous imho